custom_components/variable/sensor.py

```
@@ -160,6 +162,13 @@ def __init__(
         self._attr_native_unit_of_measurement = config.get(CONF_UNIT_OF_MEASUREMENT)
         self._attr_suggested_unit_of_measurement = None
         self._attr_state_class = config.get(CONF_STATE_CLASS)
+        self._attr_device_info = async_device_info_to_link_from_device_id(
+            hass,
+            config.get(CONF_DEVICE_ID),
+        )
+        _LOGGER.debug(
+            f"({self._attr_name}) [init] device_id: {config.get(CONF_DEVICE_ID)}, device_info: {self._attr_device_info}"
```

Ruff

Logging statement uses f-string

(G004)